installer.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
What's the benefit to run server part as a service?

Also, these changes assume that the target runs under systemd control which is not always true. It can be any type of NOS (or just plane Linux)... In fact, it does not make sense to run saivs based server on a separate device since saivs does not have dataplane (no traffic).

What I really see beneficial is to have some `run` scripts for both client and server parts.